sc/social/like/tests/test_hide_unpublished.robot

```diff
@@ -34,7 +34,6 @@ Create Document
     Click Add Document
     Input Text  css=${title_selector}  ${title}
     Click Button  Save
-    Page Should Contain  Changes saved.
```

Comment:
it looks like without **p.a.contenttypes** the text still is **Item created**, added back this check to see when it will break